MANUAL.txt

```diff
@@ -5864,6 +5864,12 @@ with the `src` attribute.  For example:
       </source>
     </audio>
 
+When using HTML as the input format (as opposed to the raw HTML
+extension), then the `data-external="1"` attribute also works the same
+as described above, but only for the `<img>` element.  The other
+elements have no representation in pandoc's AST and any attributes
+will hence be ignored.
```

Comment:
I this accurate? This was what I could determine from reading the code, but I'm not 100% sure.

Reply:
All AST elements with an [`Attr` type](https://github.com/jgm/pandoc-types/blob/master/src/Text/Pandoc/Definition.hs) can host attributes... not sure the HTML Reader passes them along only for images or all elements.... (you can try with `pandoc -f html -t native`

Reply:
As far as I can understand, the HTML reader only passes on attributes that it knows about, at least for the four tags mentioned in this section of the manual. Here's the output reading some dummy data into native format:
```
$ pandoc -f html -t native <<< '<img src="localhost" data-external="1" foo="bar"><audio attr="1">A</audio><video attr="2">B</video><source src="y" data-external="1" type="audio/mpeg">C</source>'
[Plain [Image ("",[],[("data-external","1")]) [] ("localhost","")]
,Plain [Str "A"]
,Plain [Str "B"]
,Plain [Str "C"]]
```
or, using a pandoc version without my `data-external` change:
```
$ pandoc -f html -t native <<< (same as above)
[Plain [Image ("",[],[]) [] ("localhost","")]
,Plain [Str "A"]
,Plain [Str "B"]
,Plain [Str "C"]]
```
This tells me `<audio>`, `<video>`, and `<source>` aren't representable as native AST elements, and that pandoc's HTML reader silently drops these tags. This is what I wanted to explain in the manual, in case a user wonders why only the `<img>`-tag supports `data-external="1"` when not using raw HTML. But, I'm happy to just remove this sentence if it adds any confusion. :-)